config/core/test-configs.yaml

```diff
@@ -2171,6 +2177,24 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-uboot
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-futex
```

Comment:
Failed, looks like something got stuck:
https://lava.sirena.org.uk/scheduler/job/84875

Also it seems to only be an issue with Clang builds, the GCC ones appear to be completing:
https://lava.sirena.org.uk/scheduler/job/84868

Reply:
This looks like a build issue with clang - the same thing is happening for the same test on existing platforms:

https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20220425.0/arm64/defconfig/clang-15/lab-collabora/kselftest-futex-meson-g12b-a311d-khadas-vim3.txt

and we're generally thin on kselftest results built with clang-15 on arm - if you look at 

https://storage.staging.kernelci.org/kernelci/staging-next/staging-next-20220425.0/arm64/defconfig/clang-15/lab-broonie/kselftest-alsa-meson-gxl-s905x-libretech-cc.txt

it looks an awful lot like the program hasn't actually been cross compiled when building for clang.

Reply:
Right, yes that makes sense. So I think this PR is good to go then, and we should chase up the kselftest build issue separately.